examples/web/README.md

```diff
@@ -1,6 +1,6 @@
 # Overview
 
-This example shows how to use [@opentelemetry/sdk-trace-web](https://github.com/open-telemetry/opentelemetry-js/tree/master/packages/opentelemetry-web) with different instrumentations from contrib repo in a browser.
+This example shows how to use [@opentelemetry/sdk-trace-web][] with different instrumentations from contrib repo in a browser.
```

Comment:
this now points to npm, is this intentional or it should still point to core repo just it should have appropriate link

Reply:
I think it's fine for this to point to npm since it's decorated as a package; There's a link to the repository further down the page. You can also get to the repository from the npm page pretty easily.